marketplace/release-notes/8.8.1.txt

```diff
@@ -0,0 +1,2 @@
+Update Apache Commons from 2.8.0 to 2.11.0
+Update Mendix Publish Plugin from 1.3 to 1.10
```

Comment:
We do not have to mention the Publish Plugin update, it's for internal use only. Customers do not have access to it.